APIComparer.Contracts/CompareNugetPackage.cs

```diff
@@ -2,21 +2,18 @@
 namespace APIComparer.Contracts
 {
     using NServiceBus;
-    using NuGet;
 
     public class CompareNugetPackage : ICommand
     {
-        public CompareNugetPackage(string packageId, SemanticVersion leftVersion, SemanticVersion rightVersion, string target)
+        public CompareNugetPackage(string packageId, string leftVersion, string rightVersion)
```

Comment:
It forced me to do quite a few `ToString` on the handler side to convert back. I didn't see any benefits of having it?

Not religious though, I can add it back? (Or perhaps add a new "semantic version" abstraction to avoid depending on NuGet for out DTO?)